packages/shorebird_cli/lib/src/shorebird_build_mixin.dart

```diff
@@ -277,6 +278,12 @@ mixin ShorebirdBuildMixin on ShorebirdCommand {
   /// https://github.com/shorebirdtech/shorebird/issues/1101 for more info.
   Future<void> _systemFlutterPubGet() async {
     const executable = 'flutter';
+    if (osInterface.which(executable) == null) {
```

Comment:
we really need to get rid of this mixin so we can avoid having to duplicate the tests 🤣

Reply:
Yeah, this PR is a case study re: why we shouldn't use mixins for any functionality we want to test :)